### DISCUSSION TITLE
Can I use wg-easy to access my home network remotely.

### DISCUSSION THREAD

**Discussion by lnee94** [wg-easy/wg-easy#894] [Q&A]
I have a server at my house running Ubuntu server 22.04.4 with the docker snap.
My goal is to be able to access all devices on my local network without them all be connected to the VPN.
My lan ip is 192.168.1.64 for the server.
My docker conf is.
```
docker run -d \
  --name=wg-easy \
  -e WG_HOST=domain \
  -e PASSWORD=password \
  -e WG_DEVICE=enp3s0 \
  -e UI_TRAFFIC_STATS=true \
  -v ~/.wg-easy:/etc/wireguard \
  -p 51820:51820/udp \
  -p 51821:51821/tcp \
  --cap-add=NET_ADMIN \
  --cap-add=SYS_MODULE \
  --sysctl="net.ipv4.conf.all.src_valid_mark=1" \
  --sysctl="net.ipv4.ip_forward=1" \
  --restart unless-stopped \
  weejewel/wg-easy
```

I don't think this question has been asked, and thanks for your time.

**Comment by sudo-nitz** [comment]
I think You should access devices in Your LAN network.

Remember to use IP address instead of NETBIOS name - i.e. _\\192.168.1.10_ instead of _\\MY-PC_.

If it still not working add Your subnet to AllowedIP parameter:
- in client add/edit line _AllowedIPs = 192.168.1.0/24, 0.0.0.0/0, ::/0_
- or/and in server add/edit parameter _WG_ALLOWED_IPS_ with _192.168.1.0/24, 0.0.0.0/0, ::/0_

By server I mean docker with adding line _-e WG_ALLOWED_IPS=192.168.1.0/24, 0.0.0.0/0, ::/0 \_  in Your configuration.

**Reply by alrapine** [reply]
this dont working, wg config file didnt get values from WG_ALLOWED_IPS parameter((

**Comment by ww-daniel-mora** [comment]
By bashing into the container and finding the container address I was able to figure out the host address and able to ssh into that.

```
$ docker exec -it <container_sha> bash
$ ip addr show eth2
> 275: eth2@if276: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default
    link/ether 02:42:ac:12:00:15 brd ff:ff:ff:ff:ff:ff link-netnsid 2
    inet 172.18.0.21/16 brd 172.18.255.255 scope global eth2
       valid_lft forever preferred_lft forever
```
This shows the container had the address `172.18.0.21`

ssh-ing into `172.18.0.1` from a client connected to the wireguard vpn connected me to the actual host.

But I don't know how stable those addresses are and I hope there is a way I can use the regular lan address instead. I just have not figured out how yet.

**Comment by n3f** [comment]
Thanks, I was able to find my ip this way too, but it's different than yours so I imagine stability is probably a problem.
